scripts/sysman/common.yml

```diff
@@ -211,6 +211,9 @@ etors:
     - name: DEVICE_UUID
       desc: $s_uuid_t
       version: "1.7"
+    - name: POWER_DOMAIN_EXP_PROPERTIES
```

Comment:
it would be good to define EXP after a certain value, as with Core, so that way the addition or removal of new EXP dont affect standard ones.

Reply:
added